docs/understanding_deis/components.rst

```diff
@@ -27,7 +27,9 @@ platform state. Backups and WAL logs are pushed to :ref:`Store`.
 
 Cache
 -----
-The cache is an instance of `Redis`_ used by the registry.
+The cache component is an optional instance of `Redis`_. If the cache is running,
+it will be used by the :ref:`registry`. The ``deisctl`` provisioning tool does
```

Comment:
Should we explain why one might choose to use the cache, or why it isn't used by default? It may be confusing for new users.